src/requirements/data/specializations/en.ts

```diff
@@ -12,4 +12,27 @@ const MATH2940: CollegeOrMajorRequirement = {
   slotNames: ['MATH 1910', 'MATH 1920', 'MATH 2940'],
 };
 
-export default MATH2940;
+const CHEM2080: CollegeOrMajorRequirement = {
+  name: 'Chemistry',
+  description:
+    'CHEM 2090 and CHEM 2080 (can also be substituted by CHEM 2150, PHYS 2214, PHYS 2218, BTRY 3080, ECON 3130, MATH 2930, or MATH 4710)',
+  source: 'https://www.cs.cornell.edu/undergrad/rulesandproceduresengineering/engineeringchecklist',
+  checker: includesWithSubRequirements(
+    ['CHEM 2090'],
+    [
+      'CHEM 2080',
+      'CHEM 2150',
+      'PHYS 2214',
+      'PHYS 2218',
+      'BTRY 3080',
+      'ECON 3130',
+      'MATH 2930',
+      'MATH 4710',
+    ]
+  ),
+  fulfilledBy: 'courses',
+  perSlotMinCount: [1, 1],
+  slotNames: ['CHEM 2090', 'CHEM 2080'],
```

Comment:
Another nit but could the `CHEM 2080` name be something like `CHEM 2080 (or alternative)`